Source/Charts/Data/Implementations/Standard/PieChartDataSet.swift

```diff
@@ -84,6 +84,9 @@ open class PieChartDataSet: ChartDataSet, IPieChartDataSet
     /// When valuePosition is OutsideSlice, indicates line color
     open var valueLineColor: NSUIColor? = NSUIColor.black
 
+    /// When valuePosition is OutsideSlice and enabled, line will have the same color as the slice
+    open var valueLineAutoColor: Bool = false
```

Comment:
Hi there, can you change it to `useValueColorForLine`? the name does not properly imply it's bool. If you can change it, it will be easier for me to merge this week. Otherwise I have to create another commit after yours. If you cannot see this.

Reply:
@AlexeiGitH  can you see this?

Reply:
@liuxuan30 completely missed it! 
Will make suggested update and push a new commit shortly.

Reply:
@liuxuan30 Done